Frontend/src/index.tsx

```diff
@@ -1,48 +1,38 @@
-import React, { useEffect } from "react";
-import ReactDOM from "react-dom/client";
-import "./index.css";
-import App from "./App";
-import reportWebVitals from "./reportWebVitals";
-import { ChakraProvider, useColorMode } from "@chakra-ui/react";
-import { Toaster } from "react-hot-toast";
+import { ChakraProvider, LightMode, useColorMode } from "@chakra-ui/react"
+import React, { useEffect } from "react"
+import ReactDOM from "react-dom/client"
+import { Toaster } from "react-hot-toast"
+import App from "./App"
+import Web3 from "./components/Web3"
+import "./index.css"
+import reportWebVitals from "./reportWebVitals"
 
-const root = ReactDOM.createRoot(
-  document.getElementById("root") as HTMLElement
-);
-
-function ForceLightMode(props: { children: JSX.Element }) {
-  const { colorMode, toggleColorMode } = useColorMode();
-
-  useEffect(() => {
-    if (colorMode === "light") return;
-    toggleColorMode();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [colorMode]);
-
-  return props.children;
-}
+const root = ReactDOM.createRoot(document.getElementById("root") as HTMLElement)
 
 root.render(
   <ChakraProvider>
-    <ForceLightMode>
-      <React.StrictMode>
-        <App />
-      </React.StrictMode>
-    </ForceLightMode>
-    <Toaster
-      position="bottom-center"
-      toastOptions={{
-        style: {
-          borderRadius: "1rem",
-          maxWidth: "40rem",
-          marginBottom: "2rem",
-        },
-      }}
-    />
+    <Web3>
+      <LightMode>
+        <React.StrictMode>
+          <App />
+        </React.StrictMode>
+
+        <Toaster
+          position="bottom-center"
+          toastOptions={{
+            style: {
+              borderRadius: "1rem",
+              maxWidth: "40rem",
+              marginBottom: "2rem",
+            },
+          }}
+        />
+      </LightMode>
+    </Web3>
```

Comment:
Nice. I like this Web3 component/ wrapper. Much better using wagmi, yes

Reply:
Yeah, there are many libraries for it, just found wagmi convenient